### ISSUE TITLE
Cannot set R interpreter, NPE in RSettingsConfigurable

### ISSUE THREAD

**Issue by Kipriz** [holgerbrandl/r4intellij#87]
When I try to set R interpreter, I get:
```
null
java.lang.NullPointerException
	at com.r4intellij.settings.RSettingsConfigurable.isModified(RSettingsConfigurable.java:73)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.isModified(ConfigurableWrapper.java:164)
	at com.intellij.openapi.options.newEditor.SettingsEditor.a(SettingsEditor.java:378)

```

Intellij Idea, 2017.1.1
R plugin version 1.0.3.

**Comment by holgerbrandl** [comment]
This issue was fixed already in v1.0.4. Just update your plugin.

**Comment by Kipriz** [comment]
Oh, thank you! It works now.
